### PR TITLE
Responsive fixes

### PR DESCRIPTION
**Pull request by dansydoodles**

### CODE REVIEW

File: components/about.tsx [hunk 1]
```diff
@@ -31,10 +31,10 @@ export const About = () => {
             </p>
           </div>
         </div>
-        <div className={"lg:pl-32"}>
+        <div className={"lg:px-16"}>
           <p
             className={
-              "py-2 pt-4 font-roboto text-[18px] font-[400] text-[#55517B]"
+              "py-2 md:pr-8 pt-4 font-roboto text-[18px] md:text-[18px] font-[400] text-[#55517B]"
```
Comment (erick303) [inline components/about.tsx]: use responsive text sizes such as text-xl

File: components/footer.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ export const Footer = () => {
   return (
     <div>
       <div className="font-poppins">
-        <div className={"mx-8  border-t-[1px] border-[#E4E4E4] pt-4"}>
+        <div className={"mx-8  border-t-[1px] border-[#707070] pt-4"}>
```
Comment (erick303) [inline components/footer.tsx:9]: we should define common colors in the tailwind theme

File: components/metadata.tsx [hunk 3]
```diff
@@ -8,24 +8,27 @@ import mZAR from "../images/mZAR-coin.png";
...
+        <div className={"md:py-10 lg:py-8"}>
           <div
             className={
-              " pb-8 text-center font-poppins text-3xl  text-[#DE1B55]"
+              " pb-8 lg:pb-0 text-center  font-poppins text-3xl  text-[#DE1B55]"
             }
           >
-            <h2>
-              Asset<strong className="pl-2">Metadata</strong>
+            <h2 className="font-[300]">
```
Comment (erick303) [inline components/metadata.tsx]: is there no tailwind size that matches 300?

**Review by erick303 — CHANGES_REQUESTED** [review]

File: components/metadata.tsx (hunk 3 above)
Comment (dansydoodles) [reply components/metadata.tsx]: Updated where applicable

File: components/footer.tsx (hunk 2 above)
Comment (dansydoodles) [reply components/footer.tsx:9]: Will do this later

**Review by erick303 — APPROVED** [review]

**Pull request merged**